client/src/components/loginRegisterModal.js

```diff
@@ -9,50 +9,48 @@ import Tabs from '@material-ui/core/Tabs';
 import Tab from '@material-ui/core/Tab';
 import AuthContext from '../state_management/AuthContext';
 
-
 const style = {
 	error: {
 		color: 'red'
 	},
-	dialog:{
+	dialog: {
 		padding: '10px 40px',
 		display: 'flex',
 		flexDirection: 'column',
 		justifyContent: 'center',
 		alignItems: 'center'
 	},
-	appBar:{
+	appBar: {
 		marginBottom: '30px'
 	},
-	formStyle:{
+	formStyle: {
 		display: 'flex',
 		flexDirection: 'column',
 		alignItems: 'center',
 		marginBottom: '30px',
 		width: '100%'
 	},
-	hidden:{
-		visibility: "hidden",
+	hidden: {
+		visibility: 'hidden'
 	},
-	textField:{
+	textField: {
 		width: '70%'
-	},
-}
-
+	}
+};
 
 export default function LoginRegisterModal(props) {
-
 	const authContext = useContext(AuthContext);
 
 	const [loginActive, setLoginActive] = useState(true);
 	const [userData, setUserData] = useState(null);
 	const [dialogOpen, setDialogOpen] = useState(false);
 	const [errorMsg, setErrorMsg] = useState('');
-	const [asyncStart,setAsyncStart] = useState(false);
-
+	const [asyncStart, setAsyncStart] = useState(false);
 
-	const [value, setValue] = React.useState(0);//handle tabs
-	const handleChange = (event, newValue) => {setValue(newValue);};//handle tabs
+	const [value, setValue] = useState(0); //handle tabs
+	const handleChange = (event, newValue) => {
+		setValue(newValue);
+	}; //handle tabs
```

Comment:
Can we in here also reset the forms to be blank? It's a little unusual to have the email/password persist when switching between login and register

Reply:
makes sense!